server/src/main/java/org/cloudfoundry/identity/uaa/zone/MergedZoneBrandingInformation.java

```diff
@@ -33,6 +33,14 @@ public String getProductLogo() {
         return tryGet(IdentityZoneHolder.get(), BrandingInformationSource::getProductLogo).orElse(null);
     }
 
+    public String getInlinedBase64PngLogo() {
```

Comment:
I happen to have context on this, so I think that I understand what you mean by "inlined", but I think that it wouldn't make much sense to someone out of context.

Naming is hard.

I was trying to start the 7 stages of naming with something like `getFullyQualifiedWithDataTypeForHtmlUseBase64PngLogo()`.

Reply:
We think that the existing name is sufficiently clear.